### DISCUSSION TITLE
[First Time] Lapce Setup on Alpine Linux Help

### DISCUSSION THREAD

**Discussion by andar1an** [lapce/lapce#3451] [Q&A]
I am trying to setup Lapce on Alpine. I have installed it through APK, and am having difficulty understanding if additional steps are required for syntax highlighting to work - even for default supported languages like Rust, JSON, and TOML.

I have read the tree-sitter docs, and have found plugins that require external LSP servers to be installed. Some help in understanding how to have syntax highlighting without requiring external LSP servers or dependencies would be great. I am new to tree-sitter and don't know here to find grammars, and if that will even work as the current default grammars are not working. Wondering if it is due to some glibc dependencies, but it this was in APK I am assuming it was built targeting musl.

**Comment by dzhou121** [comment]
Lapce should be able to download tree sitter grammars from https://github.com/lapce/tree-sitter-grammars automatically. 

Which version of Lapce is it showing?

**Reply by andar1an** [reply]
0.4.0

**Reply by dzhou121** [reply]
Can you please try the latest 0.4.2. we fixed lots of tree sitter issues from 0.4.1

**Reply by andar1an** [reply]
Will try in November when new alpine stable is released. Don't have time to build from source right now. Added to calendar though.
